src/Grace.hs

```diff
@@ -127,6 +132,10 @@ parser = do
                 (Options.info builtins
                     (Options.progDesc "List all built-in functions and their types")
                 )
+        <> Options.command "repl"
+                (Options.info repl
+                    (Options.progDesc "Enter a repl for Grace")
```

Comment:
Minor suggestion: I usually capitalize REPL since it's an acronym

```suggestion
                    (Options.progDesc "Enter a REPL for Grace")
```